### PR TITLE
add hint for `cdf` command

### PR DESCRIPTION
**Pull request by dolphin-wood**
🙃

### CODE REVIEW

File: .functions [hunk 1]
```diff
@@ -18,5 +18,8 @@ function la() {
 
 # cd into whatever is the forefront Finder window.
 cdf() {  # short for cdfinder
-  cd "`osascript -e 'tell app "Finder" to POSIX path of (insertion location as alias)'`"
+  CWD= "`osascript -e 'tell app "Finder" to POSIX path of (insertion location as alias)'`"
```
Comment (vincentbel) [inline .functions]: I don't think a space after variable name will work 😞 , and make `CMD` a local variable will be better.

**Review by vincentbel — CHANGES_REQUESTED** [review]

**Comment by vincentbel** [comment]
@idiotWu cool idea!

File: .functions (hunk 1 above)
Comment (dolphin-wood) [reply .functions]: Ouch! Hadn't seen the space when I was using vim 😨

File: .functions [hunk 2]
```diff
@@ -18,7 +18,7 @@ function la() {
 
 # cd into whatever is the forefront Finder window.
 cdf() {  # short for cdfinder
-  CWD= "`osascript -e 'tell app "Finder" to POSIX path of (insertion location as alias)'`"
+  local CWD="`osascript -e 'tell app "Finder" to POSIX path of (insertion location as alias)'`"
   echo "cd $(CWD)"
```
Comment (vincentbel) [inline .functions]: It said `CMD: command not found` in my terminal, may just use `$CWD` here 😆

**Review by vincentbel — CHANGES_REQUESTED** [review]

**Comment by vincentbel** [comment]
@idiotWu looks good, thanks!

**Pull request merged**